src/lib.rs

```diff
@@ -205,6 +212,15 @@ pub fn jitter<P: AsRef<Path>, W: Write>(
                 let o = origin_sampler.sample(rng);
                 let d = destination_sampler.sample(rng);
                 if o.haversine_distance(&d) >= options.min_distance_meters {
+                    if options.deduplicate_pairs {
```

Comment:
makes sense, so the next bits of code only run when `--deduplicate_pairs` is passed.